### ISSUE TITLE
Slow when expanding

### ISSUE THREAD

**Issue by georgekfoury** [apploft/APLExpandableCollectionView#15]
I have around 100 items in the collectionview with picture and text. When calling self.toggleExpansionForSectionAtIndex(recognizer.section, animated: true, withDuration: 0.5);, if there are 10,20 items in the expanded section it's fine. When it contains 100 it takes around 1.5 seconds which is laggy for the user. Any solutions? (Maybe because each time i expand, the whole collectionview loads?) Thank you!

**Comment by michaelkamphausen** [comment]
Have you tested without creating and assigning the UIImage in each cell? Loading 100 images into memory is always slow.

**Comment by georgekfoury** [comment]
Yes it is still slow, the problem is that the number of the cells is high. Is there any way not to load all of the cells on expanding? (not entering cellForRowAtIndexPath) Thank you

**Comment by michaelkamphausen** [comment]
There is no way I am aware of.
